Library/Homebrew/test/command_spec.rb

```diff
@@ -0,0 +1,96 @@
+require "command"
+
+describe Homebrew::Command do
+  it "initializes correctly" do
+    command_options = Homebrew::Command
+    command_options.initialize
+    expect(command_options.valid_options).to eq([])
+  end
+
+  it "sets valid_options correctly" do
+    command_options = Homebrew::Command
+    command_options.initialize
+    command_options.option "--bar", "go to bar" do
+      command_options.option "-foo", "do foo"
+    end
+    command_options.option "--bar1", "go to bar1"
+    expect(command_options.valid_options).to eq(
+      [{ option: "--bar", desc: "go to bar", child_options: ["-foo"] },
+       { option: "-foo", desc: "do foo", child_options: nil },
+       { option: "--bar1", desc: "go to bar1", child_options: nil }]
```

Comment:
Hello. ```brew style``` is giving the following warning here (and I can't figure out how to solve this, please help):

```
C: Put a comma after the last parameter of a multiline method call.
       { option: "--bar1", desc: "go to bar1", child_options: nil }]
```

Thanks!